output.c

```diff
@@ -75,6 +100,46 @@ output_disable(struct cg_output *output)
 	wlr_output_enable(wlr_output, false);
 	wlr_output_layout_remove(output->server->output_layout, wlr_output);
 	wlr_output_commit(wlr_output);
+
+	update_output_manager_config(output->server);
```

Comment:
We shouldn't need to call `update_output_manager_config()` here, the `wlr_output_commit()` call above will result in a commit signal being fired.